subsystem/cluster_v2_test.go

```diff
@@ -0,0 +1,127 @@
+package subsystem
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/client/installer"
+	"github.com/openshift/assisted-service/internal/host/hostutil"
+	"github.com/openshift/assisted-service/models"
+)
+
+var _ = Describe("[V2ClusterTests]", func() {
+	ctx := context.Background()
+	var clusterID strfmt.UUID
+	var infraEnvID strfmt.UUID
+	var ips []string
+	var h1, h2, h3 *models.Host
+
+	AfterEach(func() {
+		clearDB()
+	})
+
+	BeforeEach(func() {
+		clusterReq, err := userBMClient.Installer.RegisterCluster(ctx, &installer.RegisterClusterParams{
```

Comment:
We should use V2 API here also

Reply:
/hold
Need to use V2 API when creating the cluster

Reply:
fixed